plugins/colorschemes/dracula-nvim.nix

```diff
@@ -0,0 +1,19 @@
+{
+  lib,
+  pkgs,
+  ...
+}:
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "dracula";
+  originalName = "dracula.nvim ";
+  defaultPackage = pkgs.vimPlugins.dracula-nvim;
+
+  maintainers = [ lib.nixvim.maintainers.refaelsh ];
+
+  settingsExample = {
+    settings = {
+      italic_comment = true;
+      colors.green = "#00FF00";
+    };
+  };
```

Comment:
Ah, I'm an idiot. This should've not had a nested `settings` within the example.

Also it seems I missed that there were unsquashed fixup commits.